docs/adr/0003-definition-of-terms.md

```diff
@@ -0,0 +1,66 @@
+# 1. Record architecture decisions
+
+Date: 2021-03-29
+
+## Status
+
+Proposed
+
+## Context
+
+We need a fairly well understood set of terms to use in the API. The terms are as follows:
+
+- Asset
+  - An archive/binary/package that can be downloaded.
```

Comment:
Please can we also define what we mean by an archive/binary/package? Are they interchangeable names for the same thing or does each have their own unique definition?